src/main/scala/passes/Checker.scala

```diff
@@ -59,7 +59,7 @@ object Checker {
     def checkLVal(e: Expr)(implicit env: Env): Env = checkE(e)
 
     def checkC(cmd: Command)(implicit env: Env): Env = cmd match {
-      case _:CSplit | _:CView | CEmpty => env
+      case _:CSplit | _:CView | CEmpty | CDecorate(_) => env
```

Comment:
Prefer `_: CDecorate` over exact matches of parameter lists.

Reply:
Yep! Thanks.